lib/client.js

```diff
@@ -5,6 +5,9 @@ const NodeCache = require('node-cache')
 const log = require('loglevel').getLogger('nypl-data-api-client')
 const OAuth = require('oauth')
 
+const TokenExpirationError = require('./errors').TokenExpirationError
+const TokenRefreshError = require('./errors').TokenRefreshError
+
 class Client {
 
   constructor (opts) {
```

Comment:
This certainly doesn't have to happen in this PR BUT....ES6 has a syntax for default parameters that will do this for you.

```javascript
class Person {	
  constructor ({name = "stephen", age = 36} = {}) {
  	this.name = name
    this.age = age
  }
}

let me = new Person
let notMe = new Person({name: 'Jobin'})
let oldDude = new Person({age: 5000})

console.log("my name is ", me.name, "and my age is ", me.age)
//my name is  stephen and my age is  36

console.log("notMe's name is ", notMe.name, "but his age is still ", notMe.age)
//notMe's name is  Jobin but his age is still  36

console.log("Although oldDude's name is ", oldDude.name, "his age is ", oldDude.age)
//Although oldDude's name is  stephen his age is  5000
```

I don't THINK this would break your constructor's syntax since you're already accepting an object.